<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleanCallGraphWidget</class>
 <widget class="QDialog" name="CleanCallGraphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSAGUI - clean call graph</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input call graph:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="inputFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="inputFileButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>List of files for deleting from input call graph:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QListWidget" name="listOfCFiles"/>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
       </property>
       <property name="shortcut">
        <string>Del</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Output call graph:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLineEdit" name="outputFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="outputFileButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inputFileEdit</tabstop>
  <tabstop>inputFileButton</tabstop>
  <tabstop>listOfCFiles</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>delButton</tabstop>
  <tabstop>outputFileEdit</tabstop>
  <tabstop>outputFileButton</tabstop>
  <tabstop>generateButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CleanCallGraphWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
